back/src/main/java/com/speakerspace/controller/EventController.java

```diff
@@ -0,0 +1,126 @@
+package com.speakerspace.controller;
+
+import com.speakerspace.dto.EventDTO;
+import com.speakerspace.security.AuthenticationHelper;
+import com.speakerspace.service.EventService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.*;
+
+import java.nio.file.AccessDeniedException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/event")
+public class EventController {
```

Comment:
Tu as lombok donc utilise le partout (à reporter aussi sur les autres classes où ce n'est le cas)
```suggestion
@RequiredArgsConstructor
public class EventController {
```